From d55e59cff9877e744a8e6f631db1324d0bb435ff Mon Sep 17 00:00:00 2001
From: Leonardo Held <leonardo.held@toradex.com>
Date: Fri, 24 Feb 2023 16:11:29 -0300
Subject: [PATCH 1/1] Remove -WError

cpr needs to be compiled from source, but the curl version is too
new and throws out an error for curl_mime_free().

This patch can be removed once cpr catches up with curl.

Signed-off-by: Leonardo Held <leonardo.held@toradex.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d0e8854..d86a029 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -105,7 +105,7 @@ endif()
 
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
 else()
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wpedantic -Werror")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wpedantic")
 endif()
 
 # SSL
-- 
2.34.1

